notebooks/thoth_support_github_data.ipynb

```diff
@@ -0,0 +1,873 @@
+{
```

Comment:
<p>Lets add these to the Pipfile and lock</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-services-group/metrics/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>